SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+The Dockstore system routinely receives security updates to the most recently
+released tagged minor version. All previous versions are unsupported.
+
+## Reporting a Vulnerability
+
+Users are able to open helpdesk tickets on [Discourse](https://discuss.dockstore.org/). Users can create helpdesk tickets in case of privacy complaints, security vulnerabilities, or any other urgent matter related to Dockstore. Helpdesk tickets will be addressed by Dockstore administrators.
+
+The following steps can be taken to create a helpdesk ticket (also shown [here](https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506)).
+
+1. Navigate to [Discourse](https://discuss.dockstore.org/) and login.
+2. Select your profile icon, located in the top right corner of the screen.
+3. Select the `mail` icon, located in the dropdown.
+4. Send a message to the `dockstore_admins` group.
+
+Note
+
+> If you are unable to see a New Message button on the mail page, you may be considered a new user and have insufficient privileges. Entering 5 topics and viewing 30 posts over a minimum of 10 minutes will raise your privileges. You will be notified of any privilege changes to your account via the mailbox.
```

Comment:
Ugh, this is lame, but I don't what we can do about it. If I'm a brand new user to Dockstore and notice the web site has been vandalized or is showing sensitive data, I have to create 5 topics in order to notify the admins about it.

Is there a Discourse setting for this?

Reply:
I suspect that there are knobs and dials that can be tuned.
This is probably a default that balances spam vs real content.

Reply:
Yeah, it's spam prevention, specifically preventing unwanted direct messages only (since new users can post public topics). Discuss has a number of other spam prevention techniques so I think we could consider lowering this privilege. I poked around discuss and I believe the change would be to set this to `0: new user` instead of `1: basic user`.

Reply:
I think the tricky part here is I suspect (based on just the screenshot) that newly signed up spammers could spam everyone as a new users, not just the helpdesk.

Reply:
i.e. maybe we need a setting where new users can spam us but not others?

Reply:
Created an issue to investigate alternatives https://ucsc-cgl.atlassian.net/browse/SEAB-4291